### PR TITLE
chore: add metadata for filter box plugin

### PR DESCRIPTION
**Pull request by suddjian**
### SUMMARY

The title says it all.

### TESTING INSTRUCTIONS

nothing to verify except that the app builds

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js [hunk 1]
```diff
@@ -22,9 +22,12 @@ import thumbnail from './images/thumbnail.png';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Tools'),
   name: t('Filter box'),
-  description:
-    'A multi filter, multi-choice filter box to make dashboards interactive',
+  description: t(`Chart component that lets you add a custom filter UI in your dashboard. When added to dashboard, a filter box lets users specify specific values or ranges to filter charts by. The charts that each filter box is applied to can be fine tuned as well in the dashboard view.
+
```
Comment (fintzd) [inline superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js:28]: Just wondering why the extra dropped row here, am I missing something that I should not? Sorry if that's how it is supposed to be and my ignorance is at fault here!

Comment (suddjian) [reply superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js:28]: It's just there for formatting.

Comment (fintzd) [reply superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js:28]: Thanks!

File: superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js [hunk 2]
```diff
@@ -22,9 +22,12 @@ import thumbnail from './images/thumbnail.png';
...
+    Note that this plugin is being replaced with the new Filters feature that lives in the dashboard view itself. It's easier to use and has more capabilities!`),
+  tags: [t('Predictive')],
```
Comment (rusackas) [inline superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js:30]: Filterbox is not predictive... something went funky here.

Comment (suddjian) [reply superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js:30]: Oh you're right

Comment (suddjian) [reply superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js:30]:
```suggestion
```

**Review by rusackas — APPROVED** [review]

**Review by geido — APPROVED** [review]
LGTM!

**Pull request merged**